### PR TITLE
Develop skip verify

### PR DESCRIPTION
**Pull request by gianpaolof**
Hello.
Small changes:
- added docker rule in the makefile
- fixed folders

### CODE REVIEW

File: examples/aws-export/res/docker/configuration.toml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+CheckInterval = '10s'
+Host = 'localhost'
+Port = 48095
+Protocol = 'http'
+ReadMaxLimit = 100
+StartupMsg = 'AWS IoT Export Service'
+Timeout = 5000
+
+[Registry]
+Host = 'localhost'
```
[the remark this answers is not in the document]
Comment (gianpaolof) [reply examples/aws-export/res/docker/configuration.toml]: > 
> 
> Need to be the docker name, i.e. edgex-consul. Similar for the rest of `Host` names. See https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/examples/simple-filter-xml/res/docker/configuration.toml for reference.

Thank you, fixed

File: examples/aws-export/res/docker/configuration.toml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        Host = '*'
+        Port = 5564
+        Protocol = 'tcp'
+    [MessageBus.SubscribeHost]
+        Host = 'localhost'
+        Port = 5563
+        Protocol = 'tcp'
+
+[Logging]
+EnableRemote = false
```
[the remark this answers is not in the document]
Comment (gianpaolof) [reply examples/aws-export/res/docker/configuration.toml]: > 
> 
> Should be true when running from docker

fixed

File: examples/aws-export/res/docker/configuration.toml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+File = './logs/aws-export.log'
```
[the remark this answers is not in the document]
Comment (gianpaolof) [reply examples/aws-export/res/docker/configuration.toml]: fixed

File: examples/aws-export/res/docker/configuration.toml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+[Registry]
+Host = 'edgex-core-consul'
+Port = 8500
+Type = 'consul'
+
+[Clients]
+  [Clients.CoreData]
+  Protocol = 'http'
+  Host = 'localhost'
```
Comment (lenny-intel) [inline examples/aws-export/res/docker/configuration.toml]: This one also, see reference I gave you for the value

File: examples/aws-export/res/docker/configuration.toml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  Port = 48080
+  [Clients.Logging]
+  Protocol = "http"
+  Host = "localhost"
```
Comment (lenny-intel) [inline examples/aws-export/res/docker/configuration.toml]: And this one, see reference I gave you for the value

File: examples/aws-export/res/docker/configuration.toml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  Port = 48061
+
+[MessageBus]
+Type = 'zero'
+    [MessageBus.PublishHost]
+        Host = '*'
+        Port = 5564
+        Protocol = 'tcp'
+    [MessageBus.SubscribeHost]
+        Host = 'localhost'
```
Comment (lenny-intel) [inline examples/aws-export/res/docker/configuration.toml]: And this one, see reference I gave you for the value

File: examples/aws-export/res/docker/configuration.toml [hunk 7]
```diff
@@ -0,0 +1,57 @@
+[Writable]
+LogLevel = 'DEBUG'
+
+[Service]
+BootTimeout = 30000
+ClientMonitor = 15000
+CheckInterval = '10s'
+Host = 'localhost'
```
Comment (lenny-intel) [inline examples/aws-export/res/docker/configuration.toml]: All references to `localhost` must be changed to appropriate docker name.
see reference I gave you for the value

File: examples/aws-export/res/docker/configuration.toml (hunk 4 above)
Comment (gianpaolof) [reply examples/aws-export/res/docker/configuration.toml]: Sorry, should be ok now.

File: examples/aws-export/res/docker/configuration.toml (hunk 5 above)
Comment (gianpaolof) [reply examples/aws-export/res/docker/configuration.toml]: fixed

File: examples/aws-export/res/docker/configuration.toml (hunk 6 above)
Comment (gianpaolof) [reply examples/aws-export/res/docker/configuration.toml]: fixed

File: examples/aws-export/res/docker/configuration.toml (hunk 7 above)
Comment (gianpaolof) [reply examples/aws-export/res/docker/configuration.toml]: fixed

**Pull request merged**